orchagent/muxorch.cpp

```diff
@@ -753,7 +759,7 @@ MuxAclHandler::MuxAclHandler(sai_object_id_t port, string alias)
     SWSS_LOG_ENTER();
 
     // There is one handler instance per MUX port
-    string table_name = MUX_ACL_TABLE_NAME;
+    string table_name = getPlatform() == MLNX_PLATFORM_SUBSTRING ? EGRESS_TABLE_DROP : MUX_ACL_TABLE_NAME;
```

Comment:
Can we rename this function to say, `isIngressAcl`() and chose the right one? You can temporarily add MLNX to that function and later just need to modify that function.